plugins/node/opentelemetry-instrumentation-pg/src/instrumentation.ts

```diff
@@ -164,6 +176,7 @@ export class PgInstrumentation extends InstrumentationBase {
           if (typeof args[args.length - 1] === 'function') {
             // Patch ParameterQuery callback
             args[args.length - 1] = utils.patchCallback(
+              plugin._config as InstrumentationConfig & PgInstrumentationConfig,
```

Comment:
I'd stay with `PgInstrumentationConfig` unless we understand why that's necessary